.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,42 @@
+---
+name: 🐜 Bug Report
+about: Create a report to help us improve
+title: '[bug] '
```

Comment:
Minor: any need to keep the `title` template when we have labels? Seems like noise.